src/Project/Builder.ts

```diff
@@ -43,8 +43,14 @@ export class Builder {
   public build(context: vscode.ExtensionContext) {
     try {
       this.currentWorkspace = helpers.obtainWorkspaceRoot();
-    } catch (e) {
-      Balloon.error(e);
+    } catch (e: unknown) {
+      let errmsg = 'Failed to obtain workspace root';
+      if (typeof e === 'string') {
+        errmsg = e;
+      } else if (e instanceof Error) {
```

Comment:
I'll look at this tomorrow :)
This change is from a bit hurry...

Reply:
Yes, currently only `Error` is used but may change in the future.
I'll add a TODO note for this.

Reply:
@seanshpark Actually, I think we'd better to throw only `Error` type when it comes to error handling rather than raw string because `Error` type already has `string` things and other debugging information.